CrossrefPlugin.php

```diff
@@ -411,4 +414,92 @@ public function getAllowedDoiTypes(): array
     {
         return [Repo::doi()::TYPE_PUBLICATION, Repo::doi()::TYPE_ISSUE];
     }
+
+    /**
+     * Make additional validation checks against publishing requirements
+     *
+     * @param $hookName string
+     * @param $args array [
+     * @return bool
+     * @option array Validation errors already identified
+     * @option Publication The publication to validate
+     * @option Submission The submission of the publication being validated
+     * @option array The locales accepted for this object
+     * @option string The primary locale for this object
+     * ]
+     * @see PKPPublicationService::validatePublish()
+     */
+    public function validate(string $hookName, array $args): bool
+    {
+        $errors =& $args[0];
+        $submission = $args[2];
+        $context = Application::getContextDAO()->getById($submission->getData('contextId'));
+        $publication = $submission->getCurrentPublication();
+        $issueId = $publication->getData('issueId');
+        $primaryLocale = $context->getPrimaryLocale();
+        $rules = [
+            'printIssn' => ['required', 'string'],
+            'onlineIssn' => ['required', 'string'],
+            'title' => ['required', 'array'],
+            'title.' . $primaryLocale => ['required', 'string'],
+            'doi' => ['required', 'url'],
+            'journalTitle' => ['required', 'array'],
+            'journalTitle.' . $primaryLocale => ['required', 'string'],
```

Comment:
The journal title is a required field in OJS, and must be present in the journal's primary locale, so it shouldn't be possible for the two `journalTitle` rules to fail. I suggest removing them.